.github/workflows/review.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v4
 
-      - name: AI Code Reviewer
+      - name: GPTcode-reviwer
```

Comment:
The name 'GPTcode-reviwer' is inconsistent with the previous naming convention. Ensure consistency in naming across the workflow.